api/models/Prebooking.js

```diff
@@ -10,8 +10,15 @@ var ModelHelpers = require('../lib/ModelHelpers');
 var setNormalisedRelationships = (record, done) => {
   delete record.male_prebooking;
   delete record.female_prebooking;
+  delete record.male_contingency;
+  delete record.female_contingency;
+
+  if (Boolean(record.contingency)) {
```

Comment:
Improve readability, ensure we are looking for a value of true or false. 
Previously I had not set DefaultsTo: false, thus it may be null / empty string etc.
Happy to remove, if you believe this is redundant.